### PR TITLE
Add p-strip--image pattern

### PR DESCRIPTION
**Pull request by barrymcgee**
**Note:** #18 should be reviewed and merged first

## Done

- Added .p-strip--image

## Query

We had previously discussed maybe using a `data-attr` to store the image url but I now know that the attr() feature has zero browser support and is impossible without JS. My workaround is to use an inline style for the background-image rule only will setting all other styles within the stylesheet - thoughts?

### CODE REVIEW

**Comment by anthonydillon** [comment]
Looks like you need to rebase this branch with master. It contains the jekyll changes.

**Comment by barrymcgee** [comment]
@anthonydillon It's because I created this branch from my jekyll branch so I test it - hence why #18 needs to be reviewed and merged first. I'll then rebase this branch and those changes will disappear.

File: .gitignore [hunk 1]
```diff
@@ -5,3 +5,7 @@ npm-debug.log
 css/
 build/
 yarn.lock
+/_site
+/jekyll/_site/
+Gemfile.lock
+/_site/index.html
```
Comment (anthonydillon) [inline .gitignore]: Do you need to ignore this file if it's directory is ignored?

File: jekyll/_patterns/strips/image.html [hunk 2]
```diff
@@ -0,0 +1,14 @@
+---
+layout: default
+title: Strips / Image
+category: _base
```
Comment (anthonydillon) [inline jekyll/_patterns/strips/image.html]: This is a pattern

Comment (barrymcgee) [inline jekyll/_patterns/strips/image.html]: I don't mean support for data attributes in HTML, I'm talking about support for [CSS attr()](http://caniuse.com/#feat=css3-attr) which I thought we could use to set the background of the strip without asking the end user to write more CSS.

**Comment by anthonydillon** [comment]
You're right. That is surprising. I think this branch needs to be rebased so it isn't changing the jekyll stuff.

**Comment by anthonydillon** [comment]
I think this may need to go to the working group for consensus. Would you mind creating an item for WG please.

**Comment by anthonydillon** [comment]
In fact with a :+1: from me, @nottrobin and @Lukewh it seems we have a good consensus on this. I'm happy for it to be implemented in this way.

**Pull request closed without merging**